library/std/src/net/ip.rs

```diff
@@ -1042,6 +1042,20 @@ impl From<[u8; 4]> for IpAddr {
     }
 }
 
+// One shouldn't use this union directly but please use
+// `Ipv6Addr::segments` or `Ipv6Addr::octets`.
+//
+// # Safety
+//
+// This union assumes that all fields are interpreted as big endian.
+// Also, all of its field has the same size.
+#[repr(C)]
+union in6_union {
```

Comment:
Do we need the name `_union`?

Reply:
I took the same name in C sources. Now I've changed it to Ipv6Bytes which may be more idiomatic.